definitions/procedures/pulp/remove.rb

```diff
@@ -0,0 +1,100 @@
+module Procedures::Pulp
+  class Remove < ForemanMaintain::Procedure
+    metadata do
+      description 'Remove pulp2'
+      for_feature :pulp2
+    end
+
+    def sys
+      ForemanMaintain::Utils::SystemHelpers.new
+    end
+
+    def pulp_data_dirs
+      [
+        feature(:pulp2).data_dir + '/published',
+        feature(:pulp2).data_dir + '/content',
+        feature(:pulp2).data_dir + '/importers',
+        feature(:pulp2).data_dir + '/uploads'
+      ]
+    end
+
+    def pulp_packages
+      [
+        'pulp-server', 'python-pulp-streamer', 'pulp-puppet-plugins',
+        'python-pulp-rpm-common', 'python-pulp-common',
+        'pulp-selinux', 'python-pulp-oid_validation',
+        'python-pulp-puppet-common', 'python-pulp-repoauth',
+        'pulp-rpm-plugins'
+      ]
+    end
+
+    def mongo_packages
+      find_installed_packages('rh-mongodb34*').concat(
+        find_installed_packages('mongodb*')
```

Comment:
i was actually wrong originally when i said that mongodb* could still be in use, after further digging all users should be on rh-mongodb34, but some users would be on an older db disk format (which from this scripts perspective is the same)